tortoise/__init__.py

```diff
@@ -557,9 +558,17 @@ async def init(
         use_tz = config.get("use_tz", use_tz)  # type: ignore
         timezone = config.get("timezone", timezone)  # type: ignore
 
-        logger.info(
+        # Mask passwords in logs output
+        passwords = []
+        for name, info in connections_config.items():
+            if isinstance(info, str):
+                info = expand_db_url(info)
+            passwords.append(info.get('credentials', {}).get('password'))
+
+        logger.debug(
             "Tortoise-ORM startup\n    connections: %s\n    apps: %s",
-            str(connections_config),
+            re.sub(r'|'.join(passwords), lambda m: f'{m.group(0)[0:len(m.group(0)) // 3]}***', str(connections_config))
+            if all(passwords) else str(connections_config),
```

Comment:
Thats a bit hard to read for me :(
Why can't we just iterate through passwords and running .replace() function on each password from list?

Like
```
output = str(connection_config)
for password in passwords:
    output.replace(f'"{password}"', "#####")

logger.debug(...., output, ...)
```

Reply:
Yeah, that was my first idea, but I thought that compact code is better. Now I realize that `.replace()` fits better.